modules/32_power/IntC/bounds.gms

```diff
@@ -65,6 +65,8 @@ loop(regi$(p32_factorStorage(regi,"csp") < 1),
   v32_shSeEl.lo(t,regi,"csp")$(t.val > 2100) = 2;
 );
 
+*** Fix capacity to 0 for elh2VRE now that the equation q32_elh2VREcapfromTestor pushes elh2, not anymore elh2VRE, and capital costs are 1
+vm_cap.fx(t,regi,"elh2VRE",rlf) = 0;
```

Comment:
Is elh2VRE still useful by any reason or could it be completely removed from the model?

Reply:
in the current version it is not used anymore. 

I wasn't sure if in a few month somebody gets an idea why we might need it again, so I thought I don't take it out now only to reintroduce it in a few months.